### DISCUSSION TITLE
404 Error: Content Not Found.

### DISCUSSION THREAD

**Discussion by whoisdon** [community/community#69846] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In my profile, specifically in the pinned README.md file, there is support for multiple languages. When I click on one of the hyperlinks, it should theoretically take me to the page related to the language I selected. The issue lies here: previously, it used to read the file correctly and redirect to the correct README. Now, it simply returns a 'Not Found' error. Please see the images below for reference.

![image](https://github.com/community/community/assets/102386944/819dbfae-a829-45e6-99dc-f63d87532214)
![image](https://github.com/community/community/assets/102386944/73e18ded-a549-4544-8726-0964f29e4985)
![image](https://github.com/community/community/assets/102386944/144de9d8-57d0-461d-8e8f-36284e298174)
![image](https://github.com/community/community/assets/102386944/87c204ca-69c6-4d5a-82ef-e1ab1ae6d0f5)

https://github.com/community/community/assets/102386944/5d1fe564-6313-4ace-913c-ff95f6bd2de1

**Comment by thevindu-w** [comment]
You can try to create the README files directory in the root of the repo instead of the multilingual-readme/ directory, with the names like `README.es.md` (for Spanish)

**Reply by whoisdon** [reply]
Yes, that can also resolve it, but it's not a solution to my problem. I don't understand why it's not importing.

**Comment by jamesweber7** [comment]
You are using a relative path, which does work when you're looking at the file from within the whoisdon repository (`https://github.com/whoisdon/whoisdon/blob/main/README.md`). From your profile overview page, the relative path wants to take you to `https://github.com/multilingual-readme/<Language>/README.md`, which is not an actual GitHub page and has no connection to your profile. You may want to try using an absolute path: `https://github.com/whoisdon/whoisdon/blob/main/multilingual-readme/<Language>/README.md`.

**Reply by whoisdon** [reply]
Yes, your assumption is to import it as a link, and indeed, it is a functional and fast solution. However, my point is not specifically about that because I am importing the files correctly, and in my mind, this should simply be functional (as it was working correctly), but it is now presenting this error.
